Source/WebKit/Configurations/webpushd.xcconfig

```diff
@@ -66,8 +66,9 @@ LAUNCHD_PLIST_INSTALL_PATH = $(LAUNCHD_PLIST_INSTALL_PATH_$(WK_RELOCATABLE_WEBPU
 LAUNCHD_PLIST_INSTALL_PATH_YES = /System/Library/LaunchDaemons
 LAUNCHD_PLIST_INSTALL_PATH_YES[sdk=macos*] = /Library/Apple/System/Library/LaunchAgents
 
-LAUNCHD_PLIST_INSTALL_PATH_NO = /System/Library/LaunchDaemons
-LAUNCHD_PLIST_INSTALL_PATH_NO[sdk=macosx*] = /System/Library/LaunchAgents
+LAUNCHD_PLIST_INSTALL_PATH_NO = $(LAUNCHD_PLIST_INSTALL_PATH_NO_$(USE_SYSTEM_CONTENT_PATH))
+LAUNCHD_PLIST_INSTALL_PATH_NO_YES = /System/Library/LaunchDaemons
+LAUNCHD_PLIST_INSTALL_PATH_NO_YES[sdk=macos*] = $(SYSTEM_SECONDARY_CONTENT_PATH)/System/Library/LaunchAgents
```

Comment:
Seems like we are assuming USE_SYSTEM_CONTENT_PATH is always YES. Should be fine, since downlevel updates have WK_RELOCATABLE_WEBPUSHD, so they will use the path above instead of expanding this setting to the empty string.

But in that case, I think we can just drop the USE_SYSTEM_CONTENT_PATH check altogether:
```suggestion
LAUNCHD_PLIST_INSTALL_PATH_NO = /System/Library/LaunchDaemons
LAUNCHD_PLIST_INSTALL_PATH_NO[sdk=macos*] = $(SYSTEM_SECONDARY_CONTENT_PATH)/System/Library/LaunchAgents
```